tests/testthat/test_t_init.R

```diff
@@ -3,34 +3,37 @@
 ## e.g. TDM before first dose:
 ## at t=-8, conc=10000
 ## Use this as true value in the simulations
+par   <- list(CL = 7.67, V = 97.7, TDM_INIT = 500)
+mod <- new_ode_model(
+  code = "dAdt[1] = -(CL/V)*A[1]; CONC = A[1]/V",
+  state_init = "A[1] = TDM_INIT * V",
+  parameters = par,
+  obs = list(cmt = 1, scale = "V"),
+  declare_variables = c("CONC"),
+  cpp_show_code = F
+)
+reg <- new_regimen(amt = 100000, times=c(0, 24), type="bolus")
+s <- sim(
+  ode = mod,
+  parameters = par,
+  n_ind = 1,
+  regimen = reg,
+  only_obs = TRUE,
+  output_include = list(variables = TRUE),
+  t_init = 10
+)
 
 test_that("TDM before first dose is considered a true initial value", {
   skip_on_cran()
-  par   <- list(CL = 7.67, V = 97.7, TDM_INIT = 500)
```

Comment:
the skip_on_cran() line can be removed from this code chunk. the reason it is here is because models take some time to compile and we were trying to get our total time down. this test_that block no longer takes much time to run so there is no reason to skip it, unless we also want to wrap the `new_ode_model` model and related lines such that they run only if not on cran.